backend/spec/factories.rb

```diff
@@ -105,6 +104,121 @@ def JSONModel(key)
       accession_date { generate(:yyyy_mm_dd) }
     end
 
+    factory :agent_record_control, class: JSONModel(:agent_record_control) do
```

Comment:
@avatar382 I wonder if it might make sense to put all the agent factories in their own rb file?